java/tsfile/src/main/java/org/apache/tsfile/encoding/decoder/FloatDecoder.java

```diff
@@ -97,6 +99,10 @@ public FloatDecoder(TSEncoding encodingType, TSDataType dataType) {
   public float readFloat(ByteBuffer buffer) {
     readMaxPointValue(buffer);
     int value = decoder.readInt(buffer);
+    if (valueItselfOverflowInfo != null && valueItselfOverflowInfo.isMarked(position)) {
+      position++;
+      return BytesUtils.bytesToFloat(BytesUtils.intToBytes(value));
```

Comment:
May use the native methods Float.intBitsToFloat() and Float.floatToIntBits(), which are possibly more efficient.

Reply:
Fixed